latest_migrations.manifest

```diff
@@ -1,6 +1,6 @@
 admin: 0003_logentry_add_action_flag_choices
 auth: 0011_update_proxy_permissions
 contenttypes: 0002_remove_content_type_name
-posthog: 0063_team_completed_snippet_onboarding
+posthog: 0064_auto_20200624_1842
```

Comment:
Looks good. Just have to reorder the migrations and update this against master!

Reply:
Rebased :+1: